automated-tests/src/main/java/com/hp/ov/sdk/network/FcNetworkResource.java

```diff
@@ -115,4 +122,26 @@ private FcNetwork builderUpdate(FcNetwork fcNetwork) {
         return fcNetwork;
     }
 
+	public String patchFcNetwork(String id) {
+		FcNetwork fcNetwork = new FcNetwork();
+		fcNetwork.setName(resourceProperties.get("name"));
+

```

Comment:
Please fix indentation for the 2 lines above.

Reply:
Done